content/topics/data_validation/_index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Data Validation
+ready: false
+---
+
+### Data Validation
+Data validation means exactly what it sounds like, your program checks the data to make sure it meets some rules or restrictions. Validation and verification are two ways to check that the data entered into a computer is correct, because data entered incorrectly is of little use. There are many different data validation checks that can be done.
+
```

Comment:
Why is validation important?

You can use some umuzi flavoured examples. Eg: when people apply to Umuzi they sometimes provide invalid data. Eg: invalid id numbers and cellphone numbers. Often we just need to make sure that whatever information is provided is in the right shape. 

Validation is important for a lot of reasons. One big reason is that a lot of data comes from people. People make mistakes, and not everyone is data literate or even computer literate enough to consistently create good data. Data capturers make mistakes, people mess up when filling in forms, people do strange things when interacting with websites and apps.

Sometimes computers made invalid data. This could be due to software errors or even hardware faults.